src/main/java/com/karl/wrappers/Threadable.java

```diff
@@ -0,0 +1,5 @@
+package com.karl.wrappers;
+
+public interface Threadable {
+  public void start();
```

Comment:
Correct me if I'm wrong but I believe this `public` for `start()` is redundant since interface methods are public by default.